### ISSUE TITLE
wget Multiple Files

### ISSUE THREAD

**Issue by jmwright** [sabrehagen/freecad-to-step#2]
I'm reworking my KiCAD converter to be a Docker container with the same structure as this one. In the prepare stage I'm having to do this to download multiple files.
```bash
wget -i /input/input.url -P /output/
```
The freecad provider in this repo assumes one file. 

When downloading multiple files this way the question then becomes, if we don't explicitly set an output filename, how does the conversion script know what file to convert? I can hard code the sample filename for now, but at some point we'll have to figure out how to handle this.

**Comment by sabrehagen** [comment]
True, the output filename would be unknown unless you specified it explicitly. The way I'd do this is as part of the next step in the pipeline, I'd have a `readInputFiles` function that lists the contents of the `/input` directory, filters for the files it's interested in (e.g. `*.fcstd`), then operate on those files.

This way you don't need to specify an explicit filename. However, depending on your requirements, you may need to specify an exact filename. If that's the case, we can think up a solution to that.
